registrations/views.py

```diff
@@ -15,195 +16,31 @@
-        'redcross.be',
-        'redcross.bg',
-        'redcross.by',
-        'redcross.ca',
-        'redcross.ch',
-        'redcross.com.fj'
-        'redcross.dm',
-        'redcross.ee',
-        'redcross.fi',
-        'redcross.ge',
-        'redcross.gr',
-        'redcross.ie',
-        'redcross.int',
-        'redcross.is',
-        'redcross.lk',
-        'redcross.lt',
-        'redcross.lv',
-        'redcross.md',
-        'redcross.mn',
-        'redcross.no',
-        'redcross.or.ke',
-        'redcross.or.kr',
-        'redcross.or.th',
-        'redcross.org',
-        'redcross.org.au',
-        'redcross.org.ck',
-        'redcross.org.cn',
-        'redcross.org.cy',
-        'redcross.org.hk',
-        'redcross.org.jm',
-        'redcross.org.kh',
-        'redcross.org.lb',
-        'redcross.org.mk',
-        'redcross.org.mm',
-        'redcross.org.mo',
-        'redcross.org.mt',
-        'redcross.org.mz',
-        'redcross.org.na',
-        'redcross.org.nz',
-        'redcross.org.pg', # Papua New Guinea
-        'redcross.org.ph',
-        'redcross.org.pg',
-        'redcross.org.rs',
-        'redcross.org.sb', # Solomon Islands
-        'redcross.org.sg',
-        'redcross.org.ua',
-        'redcross.org.uk',
-        'redcross.org.vn',
-        'redcross.org.za',
-        'redcross.org.zm',
-        'redcross.ru',
-        'redcross.se',
-        'redcross.sk',
-        'redcross.tl',
-        'redcrossghana.org',
-        'redcrosseth.org',
-        'redcrossmuseum.ch',
-        'redcrossnigeria.org',
-        'redcrossseychelles.sc',
-        'redcrossug.org',
-        'redcrossvanuatu.com',
-        'redcrosszim.org.zw',
-        'rks.si',
-        'rmiredcross.org',
-        'rodekruis.be',
-        'roteskreuz.be',
-        'rodekors.dk',
-        'rodekruis.nl',
-        'redcross.nl',
-        'roteskreuz.at',
-        'st.roteskreuz.at', # Steiermark (Styria)
-        's.roteskreuz.at',  # Salzburg
-        'b.roteskreuz.at',  # Burgenland
-        't.roteskreuz.at',  # Tirol (Tyrol)
-        'w.roteskreuz.at',  # Wien (Vienna)
-        'v.roteskreuz.at',  # Vorarlberg
-        'k.roteskreuz.at',  # Kaernten (Carinthia)
-        'n.roteskreuz.at',  # Niederoesterreich (lower Austria)
-        'o.roteskreuz.at',  # Oberoesterreich (upper Austria)
-        'roteskreuz-innsbruck.at',
-        'roteskreuz.li',
-        'rwandaredcross.org',
-        'sarc.sy',
-        'sarc-sy.org',
-        'sierraleoneredcross.org',
-        'srcs.sd',
-        'standcom.ch',
-        'tgymj.gov.tm',
-        'tongaredcross.to',
-        'trcs.or.tz',
-        'ttrcs.org',
-        'vanuaturedcross.org',
-        'voroskereszt.hu' # no comma
+    allowed_list = [dom for dom in 
+        (
+            DomainWhitelist.objects
+                .filter(is_active=True)
+                .values_list('domain_name', flat=True)
+                .annotate(domain_name_lower=Lower('domain_name'))
+        )
```

Comment:
@GregoryHorvath We can also using filtering directly here instead of fetching all the data. Something like.
```suggestion
    is_allowed = (
        DomainWhitelist.objects
            .filter(is_active=True)
            .filter(domain_name__iendswith=domain)
            .exists()
    )
```
https://docs.djangoproject.com/en/2.2/ref/models/querysets/#iendswith

Reply:
Okies, should be simpler now: https://github.com/IFRCGo/go-api/pull/709/commits/f5126fe7b297fd01b360c376fcfeb6498b33ab88

Reply:
Sounds good. 
PS: You can use directly `iexact` instead of annotating.

https://docs.djangoproject.com/en/3.0/ref/models/querysets/#std:fieldlookup-iexact